scripts/cleanup.sh

```diff
@@ -0,0 +1,24 @@
+PS4="$PS4"
```

Comment:
In the original version, this line propagated PS4 from `net/net.sh` into the ssh command, but here this line now does nothing.